cluster-provision/k8s/1.21/manifests/nfs-csi/nfs-server.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: nfs-data
+spec:
+  storageClassName: local
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 30Gi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nfs-server
+  labels:
+    app: "nfs-server"
+    cdi.kubevirt.io/testing: ""
+spec:
+  selector:
+    matchLabels:
+      app: nfs-server
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: nfs-server
+        cdi.kubevirt.io/testing: ""
+    spec:
+      containers:
+      - image: quay.io/awels/nfs-server-alpine:12
```

Comment:
Yes, sounds like a good idea, who should I reach about this?

Reply:
For the first version I could just do a skopeo copy into the kubevirtci org. Later updates would have to be synced again. Maybe @awels has a better idea?

Reply:
awels is out for a few weeks, but skopeo copy sounds awesome. I don't see this image changing anytime,
the reason we got it to quay.io/awels was because it was only available on dockerhub
https://hub.docker.com/r/itsthenetwork/nfs-server-alpine/